package.json

```diff
@@ -29,52 +27,40 @@
   "dependencies": {
     "app-module-path": "2.2.0",
     "axios": "0.16.1",
-    "bcrypt": "1.0.2",
-    "body-parser": "1.17.1",
-    "connect-mongo": "1.3.2",
-    "cors": "2.8.3",
-    "cross-env": "5.0.5",
-    "dotenv": "4.0.0",
-    "express": "4.15.3",
-    "express-session": "1.15.5",
     "firebase": "3.6.4",
-    "grommet": "1.4.0",
+    "grommet": "^1.11.0",
     "immutability-helper": "2.1.2",
-    "loglevel": "1.4.1",
-    "mongoose": "4.9.9",
-    "morgan": "1.8.1",
-    "passport": "0.3.2",
-    "passport-local": "1.0.0",
-    "prop-types": "^15.5.10",
-    "react": "15.5.4",
-    "react-dom": "15.5.4",
-    "react-hot-loader": "3.0.0-beta.7",
+    "prop-types": "15.6.2",
+    "react": "16.4.2",
+    "react-dom": "16.4.2",
+    "react-hot-loader": "^4.3.4",
     "react-redux": "5.0.5",
-    "react-router-dom": "4.1.1",
-    "redux": "3.6.0",
-    "winston": "2.3.1"
+    "react-router-dom": "4.3.1",
+    "redux": "3.6.0"
   },
   "devDependencies": {
     "async": "2.4.0",
     "babel-core": "6.21.0",
-    "babel-loader": "6.2.10",
-    "babel-plugin-transform-object-rest-spread": "6.23.0",
-    "babel-preset-es2015": "6.18.0",
-    "babel-preset-react": "6.16.0",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "1.7.0",
+    "babel-preset-flow": "6.23.0",
+    "babel-preset-react": "6.24.1",
+    "babel-preset-stage-2": "6.24.1",
```

Comment:
What does this get us that `env` doesn't?

Reply:
Flow and arrow functions as  methods in classes. This also allows us to set initial state outside of the constructor so that we don't have to use the constructor super boilerplate.